mlserver/rockcraft.yaml

```diff
@@ -0,0 +1,128 @@
+# Based on https://github.com/SeldonIO/MLServer/blob/1.3.5/Dockerfile
+# Operator version is 1.17.1 version for this ROCK can be found in
+# https://github.com/SeldonIO/seldon-core/blob/v1.17.1/operator/config/manager/configmap.yaml
+name: mlserver
+summary: An image for Seldon MLServer Huggingface
+description: |
+  This image is used as part of the Charmed Kubeflow product.
+version: 1.3.5_20.04_1 # <upstream-version>_<base-version>_<Charmed-KF-version>
+license: Apache-2.0
+base: ubuntu:20.04
+run-user: _daemon_
+services:
+  mlserver:
+    override: replace
+    summary: "mlserver service"
+    startup: enabled
+    command: bash -c 'export PATH=/opt/conda/bin/:/opt/mlserver/.local/bin:${PATH}:/usr/bin && export PYTHONPATH=/opt/mlserver/.local/lib/python3.8/site-packages/:${PYTHONPATH} && eval $(/opt/conda/bin/conda shell.bash hook 2> /dev/null) && mlserver start ${MLSERVER_MODELS_DIR}'
+    working-dir: "/opt/mlserver"
+    environment:
+      MLSERVER_ENV_TARBALL: "/mnt/models/environment.tar.gz"
+      MLSERVER_MODELS_DIR: "/mnt/models/"
+      LD_LIBRARY_PATH: "/usr/local/nvidia/lib64:/opt/conda/lib/python3.8/site-packages/nvidia/cuda_runtime/lib:$LD_LIBRARY_PATH"
+      TRANSFORMERS_CACHE: "/opt/mlserver/.cache"
+      NUMBA_CACHE_DIR: "/opt/mlserver/.cache"
+platforms:
+  amd64:
+
+parts:
+  mlserver:
+    plugin: nil
+    source: https://github.com/SeldonIO/MLServer
+    source-type: git
+    source-tag: 1.3.5
+    build-packages:
+      - bash
+      - tar
+      - gzip
+      - python3-dev
+      - python3-setuptools
+      - python3-pip
+      #  TO-DO: Verify need for the packages below
+      # - libgomp
+      # - mesa-libGL
+      # - glib2-devel
+      # - shadow-utils
+    stage-packages:
+      - bash
+    override-build: |
+
+      mkdir -p ${CRAFT_PART_INSTALL}/opt/mlserver/dist
+      cp ${CRAFT_PART_SRC}/setup.py .
```

Comment:
i think you already have this two file in build no ?

Reply:
As I understood those needs to be when when we building wheels.

Reply:
but theses files are already present in the build step no?

Reply:
but theses files are already present in the build step no?